### ISSUE TITLE
Make tls.cert optional in connection string

### ISSUE THREAD

**Issue by DanGould** [Kixunil/tonic_lnd#24]
If the node is reachable from a certificate authority signed cert, then it's already in the root store and trusted. That's the case for https://voltage.cloud. It would be convenient to forego manual certificate configuration with rotating certs.

**Comment by Kixunil** [comment]
Yeah, I think I had such requests already. Since PKI relies on centralized authorities I'm not motivated to do it myself but would accept a reasonable PR.
